applications/serial_lte_modem/README.rst

```diff
@@ -4,7 +4,11 @@ Serial LTE modem
 ################
 
 The Serial LTE Modem (SLM) application can be used to emulate a stand-alone LTE modem on an nRF91 Series device.
-The application accepts both the modem-specific AT commands documented in the `nRF91 AT Commands Reference Guide <AT Commands Reference Guide_>`_ and proprietary AT commands documented in :ref:`SLM_AT_intro`.
+The application accepts both the modem-specific AT commands and proprietary AT commands.
+The AT commands are documentated in the following guides:
+
+* Modem-specific AT commands - `nRF91x1 AT Commands Reference Guide`_  for the nRF9161 DK and `nRF9160 AT Commands Reference Guide`_ for the nRF9160 DK
```

Comment:
Old text did not talk about DK. Are we intentionally limiting the scope of the AT commands into DKs?

This comment is for the entire PR.

Reply:
@divipillai I don't think we should talk about DKs at all, the AT commands are valid for anything with the nRF9160 or nRF91x1 SiP.

Probably we shouldn't talk about nRF9161 either, because the same document is valid also for nRF9131. To be accurate, I think it should be nRF91x1, like in the document name.

Reply:
Agree, Thanks for the suggestion. I have update the documentation accordingly (removed mention of DKs). Please check and confirm.

Reply:
Looks good. I'll resolve this comment.